test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -10001,6 +10001,59 @@ public void RestoreNetCore_WithMultipleFrameworksWithPlatformAndAssetTargetFallb
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task RestoreNetCore_MacCatalystUsingXamarinIOS_WarnsAsync()
```

Comment:
Given that we have code for *each* assets group that exists, I think our tests should do that as well. 

You'll likely need a *theory* for that